### PR TITLE
EVAKA-4608 move citizen assistance decisions to decisions page

### PR DESCRIPTION
**Pull request by jaahon**
#### Summary

Citizen's assistance decisions are moved to the decisions page together with application decisions. Easiest to test locally by making a preschool daycare decision and an assistance decision.

### CODE REVIEW

File: frontend/src/citizen-frontend/decisions/decisions-page/Decisions.tsx [hunk 1]
```diff
@@ -3,36 +3,87 @@
...
+
+import ApplicationDecision from './ApplicationDecision'
+import AssistanceDecision from './AssistanceDecision'
 
 export default React.memo(function Decisions() {
   const t = useTranslation()
+  const { children } = useContext(ChildrenContext)
   const [applicationDecisions] = useApiState(getDecisions, [])
+  const [assistanceDecisions] = useApiState(
+    () => getAssistanceNeedDecisions(),
```
Comment (akheron) [inline frontend/src/citizen-frontend/decisions/decisions-page/Decisions.tsx:32]:
```suggestion
    getAssistanceNeedDecisions,
```

File: frontend/src/citizen-frontend/decisions/decisions-page/Decisions.tsx [hunk 2]
```diff
@@ -3,36 +3,87 @@
...
+          (decisions) =>
+            decisions.flatMap(({ decisions }) =>
+              decisions.filter(applicationDecisionIsUnread)
+            ).length
+        )
+        .getOrElse(0),
+    [applicationDecisions]
+  )
+
+  const childrenWithSortedDecisions = combine(
```
Comment (akheron) [inline frontend/src/citizen-frontend/decisions/decisions-page/Decisions.tsx:51]: Add `useMemo`?

File: service/src/main/kotlin/fi/espoo/evaka/assistanceneed/decision/AssistanceNeedDecisionCitizenController.kt [hunk 3]
```diff
@@ -24,8 +24,9 @@ class AssistanceNeedDecisionCitizenController(
     private val accessControl: AccessControl,
     private val assistanceNeedDecisionService: AssistanceNeedDecisionService
 ) {
+    // TODO: remove after 7.9.2022
```
Comment (akheron) [inline service/src/main/kotlin/fi/espoo/evaka/assistanceneed/decision/AssistanceNeedDecisionCitizenController.kt:27]: Today is 8.9.2022 🤔

File: service/src/main/kotlin/fi/espoo/evaka/assistanceneed/decision/AssistanceNeedDecisionQueries.kt [hunk 4]
```diff
@@ -331,6 +331,26 @@ fun Database.Read.getAssistanceNeedDecisionsByChildIdForCitizen(
...
 }
 
+fun Database.Read.getAssistanceNeedDecisionsForCitizen(
+    guardianId: PersonId
+): List<AssistanceNeedDecisionCitizenListItem> {
+    //language=sql
+    val sql =
+        """
+        SELECT ad.id, ad.child_id, validity_period, status, decision_made, assistance_levels,
+            selected_unit selected_unit_id, unit.name selected_unit_name,
```
Comment (akheron) [inline service/src/main/kotlin/fi/espoo/evaka/assistanceneed/decision/AssistanceNeedDecisionQueries.kt:341]: I prefer `AS` for naming columns, but the existing query didn't have it so 🤷

File: frontend/src/citizen-frontend/children/ChildrenPage.tsx [hunk 5]
```diff
@@ -0,0 +1,192 @@
+// SPDX-FileCopyrightText: 2017-2022 City of Espoo
```
[the remark this answers is not in the document]
Comment (jaahon) [reply frontend/src/citizen-frontend/children/ChildrenPage.tsx:1]: I guess some rebase didn't go as intended 😄

File: service/src/main/kotlin/fi/espoo/evaka/assistanceneed/decision/AssistanceNeedDecisionCitizenController.kt (hunk 3 above)
Comment (jaahon) [reply service/src/main/kotlin/fi/espoo/evaka/assistanceneed/decision/AssistanceNeedDecisionCitizenController.kt:27]: I guess I was a bit optimistic about the schedule 😄

**Pull request merged**